docs/index.md

```diff
@@ -94,7 +94,13 @@ If you're not familiar with the terminal and prefer a graphical user interface,
 
 ## Questions and support
 
-All fastlane related questions, problems and feature ideas should be posted as [GitHub issue](https://github.com/fastlane/fastlane/issues/new). This way, other users can search existing discussions, and everybody can jump in.
+Before submitting a new GitHub issue, please make sure to
+
+- Check out [docs.fastlane.tools](https://docs.fastlane.tools)
+- Check out the README pages on [this repo](https://github.com/fastlane/fastlane)
```

Comment:
The docs is the first item, this points to the GitHub repo, as we still have tons of docs on there :)